src/include/sof/trace/trace.h

```diff
@@ -376,6 +376,31 @@ do {									\
 	trace_unused(class, id_0, id_1, format, ##__VA_ARGS__)
 #endif
 
+/* tracing from device (component, pipeline, dai, ...) */
```

Comment:
@mmaka1 Maybe adding an example of usage in the commit message would be useful. Also, are these macros supposed to be used directly by SOF core/drivers or one always needs to create wrappers like comp_err, pipe_err etc.

Reply:
From what I understand I think these will only be used from wrappers (confirmation would be good though).

Reply:
@dbaluta I re-painted parts that would be forked at the fastest pace imo, to stop maintaining unscalable solution and limit amount of code to re-work in future. Other parts could be re-named in a similar way to have unified naming across the code however it does not seem as critical as the part refactored by this patch at the moment and could be done later.